tests/src/Interop/WinRT/NativeComponent/CMakeLists.txt

```diff
@@ -28,11 +28,8 @@ else()
   message("Using Windows SDK version ${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION} for WinRT interop tests.")
 endif()
 
-set (PROGRAM_FILES_X86 "ProgramFiles(x86)")
-set (WINDOWS_KITS_DIR "$ENV{${PROGRAM_FILES_X86}}/Windows Kits/10")
-if (NOT EXISTS ${WINDOWS_KITS_DIR})
-  set (WINDOWS_KITS_DIR "$ENV{ProgramFiles}/Windows Kits/10")
-endif()
+get_filename_component(WINDOWS_KITS_DIR "[HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows Kits\\Installed Roots;KitsRoot10]" ABSOLUTE CACHE)
```

Comment:
On Windows we always run CMake from the Developer command prompt. It sets the `WindowsSdkDir` which must always be right. I would instead use that. I would prefer that to the registry.